Tests/Unit/ListingAPITests.swift

```diff
@@ -12,16 +12,26 @@ class ListingAPITests: QuickSpec {
             let listingAPI = ListingApi(environment: environment)
 
             context("when endpoint is properly configured") {
-                it("will return valid URL Request") {
+                it("will return valid URL Request for app listing") {
                     mockBundle.mockEndpoint = "http://example.com"
                     expect(listingAPI.createURLRequest()).toEventually(beAnInstanceOf(URLRequest.self))
                 }
+
+                it("will return valid URL Request for app info") {
+                    mockBundle.mockEndpoint = "http://example.com"
+                    expect(listingAPI.createURLRequest("123")).toEventually(beAnInstanceOf(URLRequest.self))
```

Comment:
Do you think we need to add an extra test case to check whether it will return URLRequest with AppID added in the request?